### PR TITLE
fix: dhcpv6 leasetime segfault loop

### PR DESCRIPTION
**Pull request by sergelogvinov**
We are trying to get an IP-address lifetime, but we do not have it yet.

```
kern:    info: [2022-05-12T06:40:28.25916042Z]: init[663]: segfault at 20 ip 000000000201d2b1 sp 000000c00148b930 error 4 in init[400000+20ae000]
kern:    info: [2022-05-12T06:40:28.26453742Z]: Code: db 75 04 31 d2 eb 03 48 8b 10 48 8b 42 18 48 8b 5a 20 48 8b 4a 28 0f 1f 00 e8 3b a6 f7 ff 48 85 db 75 04 31 c0 eb 03 48 8b 00 <48> 8b 40 20 48 89 44 24 78 44 0f 11 bc 24 a8 00 00 00 c6 44 24 77
user: warning: [2022-05-12T06:40:28.27713142Z]: [talos] operator panicked {"component": "controller-runtime", "controller": "network.OperatorSpecController", "stack": "github.com/talos-systems/talos/internal/app/machined/pkg/controllers/network.(*operatorRunState).runWithPanicHandler.func1\x5cn\x5ct/src/internal/app/machined/pkg/controllers/network/operator_spec.go:137\x5cnruntime.gopanic\x5cn\x5ct/toolchain/go/src/runtime/panic.go:838\x5cnruntime.panicmem\x5cn\x5ct/toolchain/go/src/runtime/panic.go:220\x5cnruntime.sigpanic\x5cn\x5ct/toolchain/go/src/runtime/signal_unix.go:818\x5cngithub.com/talos-systems/talos/internal/app/machined/pkg/controllers/network/operator.(*DHCP6).renew\x5cn\x5ct/src/internal/app/machined/pkg/controllers/network/operator/dhcp6.go:248\x5cngithub.com/talos-systems/talos/internal/app/machined/pkg/controllers/network/operator.(*DHCP6).Run\x5cn\x5ct/src/internal/app/machined/pkg/controllers/network/operator/dhcp6.go:75\x5cngithub.com/talos-systems/talos/internal/app/machined/pkg/controllers/network.(*operatorRunState).runWithPanicHandle...
```

---
This change is [Reviewable](https://reviewable.io/reviews/siderolabs/talos/5547)

### CODE REVIEW

File: internal/app/machined/pkg/controllers/network/operator/dhcp6.go [hunk 1]
```diff
@@ -245,7 +245,12 @@ func (d *DHCP6) renew(ctx context.Context) (time.Duration, error) {
 
 	d.parseReply(reply)
 
-	return reply.Options.OneIANA().Options.OneAddress().ValidLifetime, nil
+	options := reply.Options.OneIANA()
```
Comment (frezbo) [inline internal/app/machined/pkg/controllers/network/operator/dhcp6.go:248]: @smira let's try adding tests around these functions that might return `nil`

**Review by frezbo — APPROVED** [review]

File: internal/app/machined/pkg/controllers/network/operator/dhcp6.go (hunk 1 above)
Comment (smira) [reply internal/app/machined/pkg/controllers/network/operator/dhcp6.go:248]: I wish I knew how to do that... we can mock responses, but that would be too much mocking?

**Comment by smira** [comment]
:+1: thanks for the fix, @sergelogvinov. I'll refactor the code a bit and get it merged
